kornia/feature/loftr/loftr.py

```diff
@@ -100,7 +100,7 @@ def __init__(self, pretrained: Optional[str] = 'outdoor', config: Dict = default
             self.load_state_dict(pretrained_dict['state_dict'])
         self.eval()
 
-    def forward(self, data: dict) -> Dict[str, torch.Tensor]:
+    def forward(self, data: Dict[str, torch.Tensor]):
```

Comment:
```suggestion
    def forward(self, data: Dict[str, Tensor]):
```
you can do now as follows `from kornia.core import Tensor`

we are isolating torch imports to have a better understanding what we consume from pytorch + adding code cleaning

Reply:
thanks, i'll change that.
for testing, i'll use your suggestion.